### DISCUSSION TITLE
Segmentation fault (v. 7.3.0/7.3.1)

### DISCUSSION THREAD

**Discussion by marbyshis** [pytest-dev/pytest#11087] [Q&A]
Hi,
we are running couple of test in Azure pipeline. Everything was up and running until pytest 7.3.0/ 7.3.1 release and now we are getting Segmentation fault      (core dumped) python3 -u -m cdppipeline.cli "$@". To troubleshoot this error I added faulthandler.enabled() to one of the test, that was probably failling. With faulthandler everyting works, which is odd as it should fail and give me a more specific info about Segmentation fault. How can we troubleshoot test failure?

**Comment by RonnyPfannschmidt** [comment]
As the error disappears when extra debugging is added, it's conceivable a use after free or recounting error that's sensitive to memory layout

In case cython or something like it is used, enable debugging

Also check what happens when doing a verbose run without output captured
